examples/panel/book-recommender/rag_book_recommender.py

```diff
@@ -0,0 +1,89 @@
+import os
+import json
+import pickle
+import pandas as pd
+
+from pathlib import Path
+from openai import OpenAI
+
+client = OpenAI()
+
+file_path = os.path.join('assets', 'goodreads.csv')
+df = pd.read_csv(file_path)
+
+
+file_path = os.path.join('assets', 'title_to_description.pkl')
```

Comment:
do not use pickle files as serialization format. better to use JSON

Reply:
[Using json now](https://github.com/ploomber/doc/pull/127/files#diff-396e0215830b7ad98dc178d3fd362e3e0a577ab3509a784b915a8242e092be98R38-R50)